packages/mdc-elevation/_mixins.scss

```diff
@@ -71,27 +180,27 @@
   $penumbra-color: rgba($color, $mdc-elevation-penumbra-opacity + $opacity-boost);
   $ambient-color: rgba($color, $mdc-elevation-ambient-opacity + $opacity-boost);
 
-  @include mdc-feature-targets($feat-color) {
-    box-shadow:
-      #{"#{$umbra-z-value} #{$umbra-color}"},
-      #{"#{$penumbra-z-value} #{$penumbra-color}"},
-      #{$ambient-z-value} $ambient-color;
-  }
+  $box-shadow: (
+    #{"#{$umbra-z-value} #{$umbra-color}"},
+    #{"#{$penumbra-z-value} #{$penumbra-color}"},
+    #{$ambient-z-value} $ambient-color,
+  );
+
+  @include mdc-elevation-shadow($box-shadow, $query: $query);
 }
 
-// Returns a string that can be used as the value for a `transition` property for elevation.
-// Calling this function directly is useful in situations where a component needs to transition
-// more than one property.
-//
-// ```scss
-// .foo {
-//   transition: mdc-elevation-transition-value(), opacity 100ms ease;
-//   will-change: $mdc-elevation-property, opacity;
-// }
-// ```
-@function mdc-elevation-transition-value(
```

Comment:
Since this PR is a breaking change for `mdc-elevation-transition-value` function, let's add a BREAKING CHANGE line in your commit message when you squash & merge.

Reply:
Done!